index.html

```diff
@@ -4,7 +4,7 @@
  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
- <link rel="stylesheet" href="style.css">
+ <link rel="stylesheet" href="/style.css">
```

Comment:
- [ ] Great job with your project 💪 . Kindly make sure your pull request is descriptive, this can be done by following the guideline provided by [microverse](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/images/pull_request_desc.gif) 🙏